pkgs/development/libraries/boost/1.84.nix

```diff
@@ -0,0 +1,14 @@
+{ callPackage, fetchurl, fetchpatch, ... } @ args:
+
+callPackage ./generic.nix (args // rec {
+  version = "1.84.0";
+
+  src = fetchurl {
+    urls = [
+      "mirror://sourceforge/boost/boost_${builtins.replaceStrings ["."] ["_"] version}.tar.bz2"
+      "https://boostorg.jfrog.io/artifactory/main/release/${version}/source/boost_${builtins.replaceStrings ["."] ["_"] version}.tar.bz2"
+    ];
+    # SHA256 from http://www.boost.org/users/history/version_1_84_0.html
+    sha256 = "6478edfe2f3305127cffe8caf73ea0176c53769f4bf1585be237eb30798c3b8e";
```

Comment:
The hash equals the the one in 1.83.0

https://github.com/NixOS/nixpkgs/blob/bef4effb3d376227e10885deb0ef4529602c28bc/pkgs/development/libraries/boost/1.83.nix#L12

Please update it, since it now builds 1.83.0 by pulling the source from the cache